docker-cleanup.sh

```diff
@@ -0,0 +1,8 @@
+sudo docker kill $(docker ps -q)
```

Comment:
This file should be removed. Docker cleanup is not in any way related to layout changes.